config/secrets.yml

```diff
@@ -15,9 +17,12 @@
 # shared:
 #   api_key: a1B2c3D4e5F6
 
-# Environmental secrets are only available for that specific environment.
+# Environmental secrets are only available for that specific environment
 
 development:
+  algolia:
```

Comment:
You might consider creating a `defaults` key above to make these available across environments without duplication:

```yml
# defaults across all environments
defaults: &defaults
  algolia_search:
    api_key: "<%= ENV.fetch('ALGOLIA_API_KEY') %>"
    application_id: "<%= ENV.fetch('ALGOLIA_APPLICATION_ID') %>"

development:
  <<: *defaults

test:
  <<: *defaults

# etc.
```

Reply:
Thanks for the &defaults clue.  I couldn't figure out how to get that to work!